src/styles/blocks/price.scss

```diff
@@ -0,0 +1,24 @@
+.price {
+  display: flex;
+}
+
+.text1 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 24px;
+  margin-left: 17px;
+  color: #616070;
+}
+
+.text2 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 16px;
+  line-height: 18px;
+  margin-top: 24px;
+  margin-left: 90px;
+}
```

Comment:
The class name 'text2' should also be in kebab-case to follow the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).